README.md

```diff
@@ -2068,6 +2081,108 @@ aem {
 }
 ```
 
+Custom configuration for the particular OSGi component can be created by [OsgiFramework](src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/OsgiFramework.kt) as well:
+```kotlin
+aem {
+    tasks {
+        register("instanceSecure") {
+            doLast {
+                aem.sync(aem.publishInstances) {
+                    osgiFramework.createConfiguration("org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet", mapOf(
+                        "alias" to "/crx/server",
+                        "dav.create-absolute-uri" to true,
+                        "dav.protectedhandlers" to "org.apache.jackrabbit.server.remoting.davex.AclRemoveHandler"
+                    ))
+                }
+            }
+        }
+    }
+}
+```
+`createConfiguration` may completely overwrite the existing configuration for given PID.
+
+It is also possible to add or modify the factory OSGi methods. This can be achieved by the following:
+```kotlin
+aem {
+    tasks {
+        register("instanceSecure") {
+            doLast {
+                aem.sync(aem.publishInstances) {
+                    osgiFramework.createConfiguration("org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended", "groovy-console", mapOf(
+                        "service.ranking" to 1,
+                        "user.mapping" to arrayOf("aem-groovy-console=groovy-console-system-user")
+                    ))
+                }
+            }
+        }
+    }
+}
+```
+
+To override particular fields in OSGi configuration only, use the `updateConfiguration` method as below:
+```kotlin
+aem {
+    tasks {
+        register("instanceSecure") {
+            doLast {
+                aem.sync(aem.publishInstances) {
+                    osgiFramework.updateConfiguration("org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet", mapOf(
+                        "alias" to "/crx/server"
+                    ))
+                }
+            }
+        }
+    }
+}
+```
+
+Factory OSGi configurations are supported as well:
+```kotlin
+aem {
+    tasks {
+        register("instanceSecure") {
+            doLast {
+                aem.sync(aem.publishInstances) {
+                    osgiFramework.updateConfiguration("org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended", "groovy-console", mapOf(
+                        "service.ranking" to 1
+                    ))
+                }
+            }
+        }
+    }
+}
+```
+
+The configuration for PID can also be deleted by the following:
+```kotlin
+aem {
+    tasks {
+        register("deleteConfig") {
+            doLast {
+                aem.sync(aem.instances) {
+                    osgiFramework.deleteConfiguration("org.apache.sling.jcr.davex.impl.servlets.SlingDavExServlet")
+                }
+            }
+        }
+    }
+}
+```
+
+Factory OSGi configurations are supported as well:
+```kotlin
+aem {
+    tasks {
+        register("deleteConfig") {
```

Comment:
too general task name

Reply:
example for delete method is redundant. only note in other example that also `delete` method is available